cmd/keygenerator/main.go

```diff
@@ -407,6 +407,8 @@ func saveSliceKeys(baseFilenameTemplate string, keys []key, converter pubKeyConv
 	var file *os.File
 	var err error
 	for i, k := range keys {
+		log.Warn(hex.EncodeToString(k.pkBytes))
```

Comment:
do we need this warn?

Reply:
no, was used to generate a test data, will remove